johnsonandjohnson/Bodiless-JS
# feat(core): Alter the Main Menu in Preview Mode

## Changes

- New optional `isHidden` prop on all contextMenu buttons.
- Preview Mode now only includes the Docs, Edit, and Last Commit icons.

## Test Instructions

- Run `npm run start`
- In preview mode, we should only see Docs, Edit, and Last Commit icons.
- In edit mode, we should still see all the icons.

## Related Issues
Closes: https://github.com/johnsonandjohnson/Bodiless-JS/issues/115
